### DISCUSSION TITLE
Questions about new setup

### DISCUSSION THREAD

**Discussion by willbarfield** [vmware/build-tools-for-vmware-aria#269] [General]
Is this project something that can be used for clients (vra, vro, etc.) to support their internal development?

**Comment by willbarfield** [comment]
more details:
I'm working to setup build-tools-for-vmware-aria and vrealize dev tools (vscode).  I've got the vscode extension installed and working on building build-tools-for-vmware-aria.  I'm running into an issue building the artifact-manager; "invalid target release: 17" which I should be able to overcome.  However, I was wondering if there is an easier way to get the artifacts vs having to build the project locally.  Is someone able to point me to the right place to get the artifacts or confirm that the only way is to build the build-tools-vmware-aria project?

**Comment by Michaelpalacce** [comment]
Hey @willbarfield, yes it can.

This is an official tool from vmware by broadcom for building and pushing to Aria and such. Other tools may exist, but I personally am unaware of them.

As for building it locally, in fact BTVA is targeted to be ran inside a CI/CD pipeline and has full capabilities as such
